### PR TITLE
use composer for html2text

### PR DESCRIPTION
**Pull request by stephenwaite**
[fixes a warning](https://github.com/openemr/openemr/pull/4808#issuecomment-1005226754)

#### Changes proposed in this pull request:
move `html2text()` to composer.json

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
         "ezyang/htmlpurifier": "4.13.0",
         "google/apiclient": "2.12.0",
         "guzzlehttp/guzzle": "7.4.0",
+        "html2text/html2text": "^4.3",
```
Comment (bradymiller) [inline composer.json:22]: would pin this to exact version of 4.3.1

File: library/ajax/log_print_action_ajax.php [hunk 2]
```diff
@@ -14,16 +14,12 @@
...
     CsrfUtils::csrfNotVerified();
 }
 
-$instance = new html2text($_POST['comments']);
-$h2t = &$instance;
-$h2t->width = 0;
-$h2t->_convert(false);
-
-EventAuditLogger::instance()->newEvent("print", $_SESSION['authUser'], $_SESSION['authProvider'], 1, $h2t->get_text());
+EventAuditLogger::instance()->newEvent("print", $_SESSION['authUser'], $_SESSION['authProvider'], 1, (new Html2Text($_POST['comments']))->getText());
```
Comment (bradymiller) [inline library/ajax/log_print_action_ajax.php:25]: 2DV

**Comment by bradymiller** [comment]
@stephenwaite , looks good! only noted minor issue

File: library/ajax/log_print_action_ajax.php (hunk 2 above)
Comment (bradymiller) [reply library/ajax/log_print_action_ajax.php:25]: actually, will think will also need the following options in there:
EventAuditLogger::instance()->newEvent("print", $_SESSION['authUser'], $_SESSION['authProvider'], 1, (new Html2Text($_POST['comments']), ['do_links' => 'none', 'width' => 0])->getText());

**Comment by bradymiller** [comment]
btw, also noted another minor issue to include options array in the new Html2Text... call to mimick settings in prior code

File: composer.json (hunk 1 above)
Comment (stephenwaite) [reply composer.json:22]: good catch, I really did do this at one point but the pain of `docker cp composer.lock` with easy dev distracted me :)

File: library/ajax/log_print_action_ajax.php (hunk 2 above)
Comment (stephenwaite) [reply library/ajax/log_print_action_ajax.php]: thanks for looking up those [options](https://github.com/mtibben/html2text/blob/master/src/Html2Text.php#L215-L225)

**Review by bradymiller — APPROVED** [review]

**Pull request merged**